styles/mdn/catppuccin.user.css

```
@@ -0,0 +1,238 @@
+  --plus-code-color: saturate(@maroon, 10%);
+  --html-code-color: saturate(@red, 10%);
+  --css-code-color: saturate(@sapphire, 10%);
+  --js-code-color: saturate(@yellow, 10%);
+  --http-code-color: saturate(@green, 10%);
+  --apis-code-color: saturate(@mauve, 10%);
+  --learn-code-color: saturate(@pink, 10%);
+  --plus-mark-color: darken(@maroon, 30%);
+  --html-mark-color: darken(@red, 30%);
+  --css-mark-color: darken(@sapphire, 30%);
+  --js-mark-color: darken(@yellow, 30%);
+  --http-mark-color: darken(@green, 30%);
+  --apis-mark-color: darken(@mauve, 30%);
+  --learn-mark-color: darken(@pink, 30%);
+  --plus-accent-background-color: fadeout(@maroon, 70%);
+  --html-accent-background-color: fadeout(@red, 70%);
+  --css-accent-background-color: fadeout(@sapphire, 70%);
+  --js-accent-background-color: fadeout(@yellow, 70%);
+  --http-accent-background-color: fadeout(@green, 70%);
+  --apis-accent-background-color: fadeout(@mauve, 70%);
+  --learn-accent-background-color: fadeout(@pink, 70%);
+  --plus-accent-engage: fadeout(@maroon, 30%);
+  --html-accent-engage: fadeout(@red, 30%);
+  --css-accent-engage: fadeout(@sapphire, 30%);
+  --js-accent-engage: fadeout(@yellow, 30%);
+  --http-accent-engage: fadeout(@green, 30%);
+  --apis-accent-engage: fadeout(@mauve, 30%);
+  --learn-accent-engage: fadeout(@pink, 30%);
+  --modal-backdrop-color: fadeout(@mantle, 30%);
+  --selection-background-color: fadeout(@surface2, 40%);
+  --text-primary-red: @red;
+  --text-primary-green: @green;
+  --text-primary-blue: @blue;
+  --text-primary-yellow: @yellow;
+  --collections-link: @flamingo;
+  --collections-header: darken(@red, 30%);
+  --collections-mandala: darken(@red, 20%);
+  --collections-icon: darken(@red, 10%);
+  --updates-link: @sky;
+  --updates-header: @crust;
+  --updates-mandala: lighten(@sky, 10%);
+  --updates-icon: @sky;
+  --ai-help-link: @green;
+  --ai-help-header: @crust;
+  --ai-help-mandala: lighten(@green, 10%);
+  --ai-help-icon: @green;
+  --form-limit-color: @overlay0;
+  --form-limit-color-emphasis: @overlay1;
+  --form-invalid-color: @red;
+  --form-invalid-focus-color: @flamingo;
+  --form-invalid-focus-effect-color: fadeout(@flamingo, 80%);
+  --baseline-high-bg: hsl(hue(@green), saturation(@green), lightness(@surface0));
+  --baseline-high-engine-bg: hsl(hue(@green), saturation(@green), lightness(@base));
+  --baseline-high-img: url(/static/media/high-dark.25169d718e5bad3cf42d.svg);
+  --baseline-high-check: @green;
+  --baseline-low-bg: hsl(hue(@blue), saturation(@blue), lightness(@surface0));
+  --baseline-low-engine-bg: hsl(hue(@blue), saturation(@blue), lightness(@base));
+  --baseline-low-check: @blue;
+  --baseline-low-img: url(/static/media/low-dark.7f95ec44f36ff865af95.svg);
+  --baseline-low-pill-bg: @sapphire;
+  --baseline-low-pill-color: @base;
+  --baseline-limited-bg: @surface0;
+  --baseline-limited-engine-bg: @base;
+  --baseline-limited-img: url(/static/media/limited-dark.2af083804a2999abf515.svg);
```

These look like MDN's default images right? Should we just get rid of these?
```suggestion
  --baseline-high-check: @green;
  --baseline-low-bg: hsl(hue(@blue), saturation(@blue), lightness(@surface0));
  --baseline-low-engine-bg: hsl(hue(@blue), saturation(@blue), lightness(@base));
  --baseline-low-check: @blue;
  --baseline-low-pill-bg: @sapphire;
  --baseline-low-pill-color: @base;
  --baseline-limited-bg: @surface0;
  --baseline-limited-engine-bg: @base;
```

---

I was thinking of making custom variant of those images, but yeah we could just remove them for now.

---

Oh yeah gotcha sorry.